common/characters/INS.txt

```diff
@@ -1184,6 +1205,55 @@ characters = {
 		}
 	}
 	
+	INS_tan_malaka = {
+		name = INS_tan_malaka
+		portraits = {
+			civilian = {
+				large = GFX_portrait_indonesia_tan_malaka
+				small = GFX_idea_indonesia_tan_malaka_civ
+			}
+			army = {
+				large = GFX_portrait_indonesia_tan_malaka_army
+				small = GFX_idea_indonesia_tan_malaka_mil
+			}
+		}
+		country_leader = {
+			ideology = leninism
+			desc = INS_tan_malaka_desc
+			traits = {
+				marxist_academic
+				dislikes_stalin
+			}
+			expire = "1965.1.1.1"
+			id = -1
+		}
+		advisor = {
+			slot = political_advisor
+			idea_token = INS_tan_malaka
+			allowed = {
+				original_tag = INS
+				has_dlc = "Man the Guns"
```

Comment:
Doesn't seem necessary? His trait doesn't affect anything MtG related

Tan Malaka has also been added on the Philippines in SEA, you could instance this character if you like the DLC portrait more